pom.xml

```diff
@@ -138,6 +133,30 @@
       <version>2.30</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
What was puzzling me was I was not seeing any **new** references to classes of those projects but the dependencies are new. They were being brought from the 1.565 parent pom I presume, sorry!